README.md

```diff
@@ -360,6 +360,7 @@ The following descriptions provide additional elaboration on a few common parame
 | `--all`  |  It's common for a big package to have a lot of interfaces, so mockery provides `--all`. This option will tell mockery to scan all files under the directory named by `--dir` ("." by default) and generates mocks for any interfaces it finds. This option implies `--recursive=true`. |
 | `--recursive`  |  Use the `--recursive` option to search subdirectories for the interface(s). This option is only compatible with `--name`. The `--all` option implies `--recursive=true`. |
 | `--output` | mockery always generates files with the package `mocks` to keep things clean and simple. You can control which mocks directory is used by using `--output`, which defaults to `./mocks`. |
+|`--outpkg`| mockery always generates files with the package `mocks` to keep things clean and simple. You can control the package name by using `--outpkg`, which defaults to `mocks`.|
```

Comment:
I would simplify this to say simply "specify the `package` name of the generated mocks".

Reply:
Got it, done!